src/components/ExtractorResponseEditor/ExtractorResponseEditor.tsx

```diff
@@ -100,6 +100,11 @@ class ExtractorResponseEditor extends React.Component<Props, State> {
                     }
 
                     this.props.onChangeCustomEntities([...nextProps.customEntities, newCustomEntity])
+
+                    // EntityPicker stays open after creation of new entity. Force it to close.
+                    this.setState({
+                        isMenuVisible: false
+                    })
```

Comment:
Feels like a cleaner way to do this is closing it when the entity creator opens.  Doing it in the 
Replace this:
` onClickNewEntity={this.props.onClickNewEntity}`

This this:
`onClickNewEntity={this.onClickNewEntity}`

And add:
 ```
 @OF.autobind
    onClickNewEntity(entityTypeFilter: string) {
        this.setState({isMenuVisible: false})
        this.props.onClickNewEntity(entityTypeFilter)
    }
```